sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregationIterator.scala

```diff
@@ -334,7 +334,7 @@ class TungstenAggregationIterator(
   // This is the hash map used for hash-based aggregation. It is backed by an
   // UnsafeFixedWidthAggregationMap and it is used to store
   // all groups and their corresponding aggregation buffers for hash-based aggregation.
-  private[this] val hashMap = new UnsafeFixedWidthAggregationMap(
+  private[aggregate] val hashMap = new UnsafeFixedWidthAggregationMap(
```

Comment:
If we change it to `private[aggregate]`, looks like we will lose the direct field access of `hashMap` (provided by `private[this]`). How about we create a method to return `hashMap` instead of changing the scope.